.github/workflows/upgrade-python-requirements.yml

```diff
@@ -17,8 +17,8 @@ jobs:
       branch: ${{ github.event.inputs.branch || github.event.repository.default_branch }}
       # optional parameters below; fill in if you'd like github or email notifications
       # user_reviewers: ""
-      # team_reviewers: ""
-      # email_address: ""
+      team_reviewers: "axim-aximprovements"
+      email_address: "aximimprovements@axim.org"
       # send_success_notification: false
```

Comment:
We can keep send_success_notification to false to avoid weekly success emails from many repositories.

Reply:
Done